src/watchmaker/workers/salt.py

```diff
@@ -720,6 +720,7 @@ def __init__(self, *args, **kwargs):
         self.bootstrap_source = \
             kwargs.pop('bootstrap_source', None) or ''
         self.git_repo = kwargs.pop('git_repo', None) or ''
+        self.salt_content_path = kwargs.pop('salt_content_path', None) or ''
```

Comment:
This seems unrelated to the rest of the changes. It wasn't needed before, so I don't think it is needed?

```suggestion
```